x-pack/plugins/cloud_security_posture/server/create_indices/latest_indices.ts

```diff
@@ -26,8 +25,8 @@ export const latestIndexConfigs: LatestIndexConfig = {
   },
   vulnerabilities: {
     indexName: VULNERABILITIES_INDEX_NAME,
-    indexPattern: LATEST_VULNERABILITIES_INDEX_PATTERN,
+    indexPattern: CDR_LATEST_NATIVE_VULNERABILITIES_INDEX_PATTERN,
     indexTemplateName: LATEST_VULNERABILITIES_INDEX_TEMPLATE_NAME,
-    indexDefaultName: LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
+    indexDefaultName: CDR_LATEST_NATIVE_VULNERABILITIES_INDEX_PATTERN,
```

Comment:
this is the confusion I mentioned - the findings configs are a bit different now, even though logically it's the same

Reply:
yes, what I meant is that misconfigurations are still following the old approach of two different constants with the same value. Not a big deal, but I though it would be more straightforward to do this change consistently for both misconfigs and vulnerabilities either in this PR or in a separate follow-up PR. Otherwise we might end up in the situation of slightly different code approach for exactly the same business logic.

Reply:
It really confused me while I was working on this change, I prefer to leave the fix on this pr.